glean/metrics.yaml

```
@@ -1637,3 +1637,179 @@ sample:
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - mcleinman@mozilla.com
+    expires: never
+    extra_keys:
+      speed:
+        description: |
+          The speed bucket that the test resulted in: Slow | Medium | Fast
+        type: string
+
+  bottom_navigation_bar_click:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      The user has tapped a bottom navigation bar button.
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-3630
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/5702#issuecomment-1397761161
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - mcleinman@mozilla.com
+    expires: never
+    extra_keys:
+      bar_button:
+        description: |
+          The name of the bar button tapped:
+          ScreenMessaging | ScreenHome | ScreenSettings | Unknown
+        type: string
+
+  authentication_error:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      The user is shown an error while authenticating in the app.
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-3630
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/5702#issuecomment-1397761161
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - mcleinman@mozilla.com
+    expires: never
+    extra_keys:
+      reason:
+        description: |
+          The error shown: Timeout | IncorrectPassword | InvalidEmail |
+          InvalidOrExpiredVerificiationCode | InvalidTOTP | InvalidUnblockCode
+        type: string
+
+  user_changed_endpoint_geo:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      User changed the geography (city, or cities if multihop) of the endpoint
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-3630
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/5702#issuecomment-1397761161
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - mcleinman@mozilla.com
+    expires: never
+    extra_keys:
+      server:
+        description: |
+          Which server changed:
+          multiHopEntryServer | multiHopExitServer | singleHopServer
+        type: string
+
+  help_menu_help_center_opened:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      The user selected the help center option within the help menu
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-3630
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/5702#issuecomment-1397761161
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - mcleinman@mozilla.com
+    expires: never
+
+  help_contact_support_opened:
```

nit:
```suggestion
  help_menu_contact_support_opened:
```
For standardization sake.

---

That was my original dream, but when trying that glean gave an error: `'help_menu_contact_support_opened' is too long`

I could pull "menu" from all the others, but then `help_menu_help_center_opened` looks really weird... I'm open to feedback on this, the lack of consistency hurts me inside as well.

---

Hum, I was afraid of that. Sadly we can't use categories for metrics unless we are confortable having them only on the new Glean. Otherwise, you would be able to have a help_menu category and each on these events inside it. That would be :chefs-kiss.

---

I'd have a preference towards not requiring new glean, to be able to collect on iOS. I know iOS changes are coming to use new glean, but it's possible/likely that it will be a release or so behind this work.

That said, don't feel strongly about that - let me know if you want me to move this to new glean and add a category.

---

Yes I agree about not requiring new Glean.